### ISSUE TITLE
[SUGGESTION]: Have Safari Zone provide party exp

### ISSUE THREAD

**Issue by umbralOptimatum** [pokeclicker/pokeclicker#4801]
### Current Behavior

The Safaris currently progress egg hatching by walking, but do not award experience to party members. If spending long periods of time in a safari with hatchery helpers active, this leads to many party members all being level 1 simultaneously.

### Improved Behavior

I think that the Safaris should award exp for catching pokemon. While this is not strictly accurate to the main games' Safari Zones, it's a better fit for Pokeclicker's mechanics more strongly emphasizing breeding and fully leveling party members.

It would also avoid awkwardness from the breeding design generally assuming the party is leveling up at the same time eggs are being hatched. Having many party members at level 1 can create breeding inefficiency if every desired filtered-for mon is stuck at a low level. When sorting by breeding efficiency, enough time in the safari could also cause potential inconvenience if every high-attack party member ends up at level 1—fighting enough enemies to relevel them would take much longer than normal with party attack slashed.

While Safari encounters don't need to be an efficient source of exp, giving _some_ would avoid the awkward and (I think) otherwise-impossible situation of having many level 1 pokemon at once.

### Detailed Description

To reward hunting rare or difficult-to-catch pokemon, the exp granted could scale inversely with catch chance and encounter weight.

**Comment by xslk** [comment]
I like how it provides no XP now so that I can end battle frontier earlier than 27 hours by getting my pokemon back to level 1 in the safari

**Comment by xslk** [comment]
If you were part of any of the discord conversations, you would see that there is more and more discussion about bringing more things closer to game canon so this could easily be rejected for not canon. 

Also, what would be the determination for rare or hard to catch pokemon? Some legendaries and mythicals have a higher catch rate then some common pokemon.

**Comment by EatPant2nd** [comment]
I also don't want Safari to give exp. There are many sources of exp outside Safari and like sLk said right now it's the only place where you can hatch Pokémon and keep them at level 1 to lower your damage. Whether this is "intended" or not, I like having the option to use this place for that purpose. It's also not a "free" thing to do as I have to spend Quest Points to enter and time to properly do this.

**Comment by TopPercentageRattata** [comment]
> If you were part of any of the discord conversations, you would see that there is more and more discussion about bringing more things closer to game canon so this could easily be rejected for not canon.

In terms of canon, catching pokemon does give experience since 2013.

**Comment by xslk** [comment]
> > If you were part of any of the discord conversations, you would see that there is more and more discussion about bringing more things closer to game canon so this could easily be rejected for not canon.
> 
> In terms of canon, catching pokemon does give experience since 2013.
> 

In the safari zone?

**Comment by TopPercentageRattata** [comment]
> > > If you were part of any of the discord conversations, you would see that there is more and more discussion about bringing more things closer to game canon so this could easily be rejected for not canon.
> > 
> > 
> > In terms of canon, catching pokemon does give experience since 2013.
> 
> In the safari zone?

Yes, in all safari zones since then. So Friend Safari in X/Y, Hoenn Safari Zone in ORAS, and GO Park in Let's Go all give experience upon capturing. 
I don't believe Great Marsh in BDSP does, but BDSP shouldn't exist anyways

**Comment by xslk** [comment]
> > > > If you were part of any of the discord conversations, you would see that there is more and more discussion about bringing more things closer to game canon so this could easily be rejected for not canon.
> > > 
> > > 
> > > In terms of canon, catching pokemon does give experience since 2013.
> > 
> > In the safari zone?
> 
> Yes, in all safari zones since then. So Friend Safari in X/Y, Hoenn Safari Zone in ORAS, and GO Park in Let's Go all give experience upon capturing. 
> I don't believe Great Marsh in BDSP does, but BDSP shouldn't exist anyways


The original safari in r/b/y, fr/lg and r/s/e don't bring you into a battle with your pokemon. Great marsh also follows the original safari formula from gen 1 and 3 in both dpp and bdsp. 

Friend safari in x/y and hoenn safari in oras both turn the safari into battle areas so that granting experience makes sense. Friend safari would need an entire revamp and be changed to a route or dungeon for this to make sense, which would be a whole other mess.

Let's go doesn't have wild battles and has an entirely different mechanic that doesn't translate to clicker.
